readme.md

```diff
@@ -22,3 +22,25 @@ serve({
   404: () => new Response("not found"),
 });
 ```
+
+### `jsx()`
+
+render with `preact` to create html Response
+
+```js
+import {
+  jsx,
+  serve,
+} from "https://deno.land/x/sift@0.1.7/mod.ts";
+import { html } from "https://cdn.skypack.dev/htm@v3.0.4/preact/standalone.module.js";
+
+const App = html`
```

Comment:
Deno Deploy supports JSX/TSX out of the box. The file extensions need to be either `.jsx/tsx` for Deno Deploy to process the code. So we don't need to use the `htm` package.

Reply:
Is my fix okay?